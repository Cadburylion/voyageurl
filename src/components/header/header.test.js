import React from 'react';
import { shallow } from 'enzyme';
import Auth from '../../auth/auth.js';
import Header from './index';

describe('Header component', () => {
  it('renders', () => {
    const auth = new Auth();
    const wrapper = shallow(<Header auth={auth} />);
    expect(wrapper).toMatchSnapshot();
  });
});
