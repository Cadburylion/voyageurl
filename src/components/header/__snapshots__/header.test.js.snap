// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component renders 1`] = `
<header__StyledHeader>
  <header__Content>
    <header__HeaderLeft>
      <header__StyledLogo />
      <header__Title>
        Voyageurl
      </header__Title>
    </header__HeaderLeft>
    <header__HeaderRight>
      <Dropdown
        title="User Menu"
      >
        <header__StyledLink
          to="/"
        >
          Home
        </header__StyledLink>
        <header__StyledLink
          to="/page1"
        >
          Page 1
        </header__StyledLink>
        <header__StyledLink
          to="/page2"
        >
          Page 2
        </header__StyledLink>
        <header__Div
          onClick={[Function]}
        >
          Logout
        </header__Div>
      </Dropdown>
    </header__HeaderRight>
  </header__Content>
</header__StyledHeader>
`;
